lib/npg_pipeline/archive/file/logs.pm

```diff
@@ -0,0 +1,128 @@
+package npg_pipeline::archive::file::logs;
+
+use Moose;
+use Carp;
+use English qw{-no_match_vars};
+
+extends qw{npg_pipeline::base};
+
+our $VERSION = '0';
+
+has 'irods_root' => ( isa => 'Str',
+                      is  => 'rw',
+                    );
+
+sub submit_to_lsf {
+  my ($self, $arg_refs) = @_;
+  my $job_sub = $self->_generate_bsub_command($arg_refs);
+  my $job_id = $self->submit_bsub_command($job_sub);;
+  return ($job_id);
+}
+
+sub _generate_bsub_command {
+  my ($self, $arg_refs) = @_;
+
+  my $irodsinstance = $self->function_list() =~ /gclp/ismx ? q(gclp) : q();
+  my $id_run = $self->id_run();
+
+  my $required_job_completion = $arg_refs->{'required_job_completion'};
+  my $timestamp = $self->timestamp();
+  my $archive_script = q{irods_log_loader.pl};
```

Comment:
Is it possible to rename the new script so that it starts with npg_? Comes handy these days when everythign is deployed to the same bin. Also the name suggests that irods logs are loaded, while it's pipelines (and some illumina?) logs are loaded to irods. Commenting here since no way to comment in SVN